src/sass/styles.scss

```diff
@@ -0,0 +1 @@
+@import  'utilities/utilities', 'variables', 'tools/tools', 'base', 'layout/layout', 'components/components';
```

Comment:
Note that you're importing a lot of CSS here that you don't use (`tools/tools`, `components/components`, also possibly `layout/layouts`?). For the end user, this means they're downloading CSS that never gets used in the site; for your instructor, this means it's hard for me to know which bits of the Sass to evaluate/focus on.